build/Containerfile.builder

```
@@ -0,0 +1,28 @@
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM registry.access.redhat.com/ubi10/nodejs-22-minimal:10.1-1764649415
```

RuboCop (1.81.7)

[fatal] unexpected token tCOLON
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)